back/engines/commercial/content_builder/spec/services/content_builder/craftjs/multilocs_in_visual_order_spec.rb

```diff
@@ -0,0 +1,167 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ContentBuilder::Craftjs::MultilocsInVisualOrder do
+  subject(:service) { described_class.new(json, with_metadata: with_metadata) }
+
+  def load_fixture(file_name)
```

Comment:
Nit: Why not just `File.read('spec/fixtures/example_description_craftjs.json')`?